test.js

```diff
@@ -0,0 +1,44 @@
+const { chromium } = require("playwright");
+
+async function runTest() {
+  // Launch the browser
+  const browser = await chromium.launch();
+
+  // Create a new browser context
+  const context = await browser.newContext();
+
+  // Create a new page
+  const page = await context.newPage();
+
+  // Navigate to the landing page
+  await page.goto("http://localhost:3000/");
+
+  //Title
+  try {
+    await page.waitForSelector("#title", { timeout: 5000 });
```

Comment:
This is not wrong, but we can improve it.

instead of using selector, it's always better to check by actual content. To avoid implementation details.

You can write 

`await expect(page).toHaveTitle(/Redesign your home/i);`